src/Components/CreateNewOption.tsx

```diff
@@ -12,87 +15,31 @@ export const CreateNewOption = () => {
 
   return (
     <div>
-      <h3>Create New Option</h3>
-      <Toggles>
-        <ToggleBtn onClick={() => setTag('Ans')} id={tag == 'Ans' ? 'AnsAct' : 'Ans'}>
-          <strong>Answer</strong>
-        </ToggleBtn>
-        <ToggleBtn onClick={() => setTag('Dist')} id={tag == 'Dist' ? 'DistAct' : 'Dist'}>
-          <strong>Distractor</strong>
-        </ToggleBtn>
-      </Toggles>
-      <div>
-        <Input type="text" placeholder="Suggest an answer or distractor for this question" />
-        <CategoryInput getCategory={getCategory} />
-        <SubmitBtn disabled>Submit</SubmitBtn>
-      </div>
+      <Container>
+        <div>
+          <Label text="Create New Option" color="blue" size={0} />
+          <TagBtn onClick={() => setTag('Ans')} id={tag == 'Ans' ? 'AnsAct' : 'Ans'} type="Answer" />
```

Comment:
if tag has two possible values only, it is better to use boolean.
```typescript
const [isAns, setIsAns] = useState(true)
```

Reply:
Maybe the tag will be 4 names, so I will check and if it is possible, then I will replace it.